curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md

```diff
@@ -22,18 +22,82 @@ Start this project on Repl.it using <a href='https://repl.it/github/freeCodeCamp
 
 ```yml
 tests:
-  - text: I can provide my own project, not the example URL.
+  - text: You should provide your own project, not the example URL.
     testString: |
       getUserInput => {
         assert(!/.*\/url-shortener-microservice\.freecodecamp\.rocks/.test(getUserInput('url')));
       }
-  - text: I can pass a URL as a parameter and I will receive a shortened URL in the JSON response.
-    testString: ''
-  - text: 'If I pass an invalid URL that doesn''t follow the valid http://www.example.com format, the JSON response will contain an error instead.'
-    testString: ''
-  - text: 'When I visit that shortened URL, it will redirect me to my original link.'
-    testString: ''
 
+  - text: "You can POST a URL to `/api/shorturl/new` and get a JSON response with `original_url` and `short_url` properties. Here's an example: `{ original_url : 'https://freeCodeCamp.org', short_url : 1}`"
+    testString: "async getUserInput => {
+      const url = getUserInput('url');
+      const urlVariable = Date.now();
+      const res = await fetch(url + '/api/shorturl/new/', {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+        body: `url=https://timestamp-microservice.freecodecamp.rocks/${urlVariable}`
```

Comment:
@scissorsneedfoodtoo Should this not be: 
```
url=https://timestamp-microservice.freecodecamp.rocks/api/timestamp/${urlVariable}
```

Reply:
Totally missed that. Thank you for catching this and submitting a fix, @Sky020. I'll review it now.